pkgs/servers/photoprism/generate-dependencies.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/tarball/21.11 -i bash -p nodePackages.node2nix curl
+
+version="220302-0059f429"
+base_url="https://raw.githubusercontent.com/photoprism/photoprism/${version}/frontend"
+
+curl "${base_url}/package.json" > package.json
```

Comment:
We don't need this, do we?

Reply:
When bumping, won't we need to fetch the `package.json` in case it's updated?

Reply:
What for? It's not use by `node2nix`. Unless you change to using `nodePackage`, in which case `package-lock.json` isn't needed

Reply:
As of now, the build fails when `package.json` isn't present:
```
error: builder for '/nix/store/mw2gxnv9f93h4wvd558jfzpll4dsm58l-photoprism-package-json.drv' failed with exit code 1;
       last 9 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/ssasc05bpqims7xn451svja8c9vmjqiz-photoprism
       > source root is photoprism
       > patching sources
       > configuring
       > no configure script, doing nothing
       > installing
       > cp: missing destination file operand after '/nix/store/5pnlxfhn3lrxyw1j12zpr1dfvmq08xf7-photoprism-package-json'
       > Try 'cp --help' for more information.
       For full logs, run 'nix log /nix/store/mw2gxnv9f93h4wvd558jfzpll4dsm58l-photoprism-package-json.drv'.
error: 1 dependencies of derivation '/nix/store/zwsjksbj8ylrfacb2ydp7aw9i9yrwsvc-node-dependencies-photoprism-1.0.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/yr9r2idbcpglk6f98k6piqdpf3anlx7g-photoprism-frontend.drv' failed to build
error: 1 dependencies of derivation '/nix/store/kmhvrgmy7jvdx0sy095py8dhdm6yy6d2-photoprism-220302-0059f429.drv' failed to build
```